plume-models/src/posts.rs

```diff
@@ -131,13 +128,15 @@ impl<'a> Provider<(&'a Connection, &'a Worker, &'a Searcher, Option<i32>)> for P
         }
 
         query
-            .get_results::<Post>(*conn)
+            .get_results::<Post>(conn)
             .map(|ps| {
                 ps.into_iter()
                     .filter(|p| {
                         p.published
-                            || user_id
-                                .map(|u| p.is_author(conn, u).unwrap_or(false))
+                            || rockets
+                                .user
+                                .clone()
```

Comment:
I think we should try to chase clones everywhere, a lot of theme could be as_ref, which more efficient in term of both memory and computation